src/servers/src/prom_row_builder.rs

```diff
@@ -209,9 +230,10 @@ mod tests {
                 value: 0.0,
                 timestamp: 0,
             }],
+            true,
```

Comment:
nit: usually boolean arguments without any hints consume quite a lot of brain power 😄. Defining `strict_mode` in the start of the test would improve readability